src/components/posts/PostsPage.js

```diff
@@ -1,54 +1,109 @@
 import { useState, useEffect } from 'react';
 import PostForm from './PostForm';
 import client from '../../utils/client';
+import Posts from './Posts';
 import './style.css';
 
 import Header from '../Header/Header';
 
 const PostsPage = () => {
-  const [post, setPost] = useState({ content: '' });
-  const [postResponse, setPostResponse] = useState('');
-  const [posts, setPosts] = useState([]);
-
-  useEffect(() => {
-    client.get('/posts').then((res) => setPosts(res.data.data.posts));
-  }, []);
-
-  const createPost = async (event) => {
-    event.preventDefault();
-    client
-      .post('/post', post)
-      .then((res) => setPostResponse(res.data))
-      .catch((data) => {
-        console.log(data);
-      });
-  };
-
-  const handleChange = (event) => {
-    event.preventDefault();
-    const { value, name } = event.target;
-    setPost({
-      ...post,
-      [name]: value,
-    });
-  };
-
-  return (
-    <>
-      <Header companyName={`Cohort Manager 2.0`} />
-      <section className='posts-section'>
-        <p>Status: {postResponse.status}</p>
-        <PostForm handleSubmit={createPost} handleChange={handleChange} />
-        <ul className='posts-list'>
-          {posts.map((post, index) => (
-            <li key={index} className='post-item'>
-              {post.content}
-            </li>
-          ))}
-        </ul>
-      </section>
-    </>
-  );
+	const [post, setPost] = useState({ content: '' });
+	const [postResponse, setPostResponse] = useState('');
+	const [posts, setPosts] = useState([]);
+	const [comment, setComment] = useState({ content: '' });
+
+	useEffect(() => {
+		client.get('/posts').then((res) => {
+			setPosts(res.data.data.posts);
+		});
+	}, []);
+
+	const createPost = async (event) => {
+		event.preventDefault();
+		client
+			.post('/post', post)
+			.then((res) => {
+				setPostResponse(res.data);
+				client.get('/posts').then((res) => {
+					setPosts(res.data.data.posts);
+				});
+			})
+			.catch((data) => {
+				console.log(data);
```

Comment:
Can we change this to be `console.error` so it displays as a red error message in the console